JavaScript/Update_README_With_Hashnode_Blogs/update_readme.js

```diff
@@ -0,0 +1,44 @@
+const fetch = require("node-fetch");
+const fs = require("fs");
+const query = `
+    {
+      user(username: "<your Hashnode username>") {
+        publication {
+          posts{
+            slug
+            title
+          }
+        }
+      }
+    }
+  `;
+
+const fetchPosts = async () => {
+  const result = await fetch("https://api.hashnode.com", {
+    method: "POST",
+    headers: {
+      "Content-type": "application/json",
+    },
+    body: JSON.stringify({ query }),
+  });
+  const response = await result.json();
+  const posts = response.data.user.publication.posts.slice(0, 3);
```

Comment:
Instead, you can take a paramter named `numPosts` in the fetchPosts function and replace `3` with `numPosts + 1 `. By this, the user can easily implement this script.

Also, `postsToShow` must be dynamic based on number of posts. A simple for loop or a `forEach` loop can do the trick!

Reply:
Best way is to abstract this piece out into another function; which generates the final posts' string. Can make the code readable to a greater extent!

Reply:
Okay. :smile:

Reply:
> replace `3` with `numPosts + 1 `. By this, the user can easily implement this script.

Isn't It just `numPosts`?

Reply:
This is just an example ;) yeah, since you are starting from 0, numpost is suitable here.

Reply:
Oh :sweat_smile: